test/sanity/validate-modules/main.py

```diff
@@ -1226,16 +1226,17 @@ def _validate_argument_spec(self, docs, spec, kwargs):
                 self.reporter.error(
                     path=self.object_path,
                     code=327,
-                    msg=('Default value from the documentation (%r) is not compatible '
-                         'with type %r defined in the argument_spec' % (doc_options_arg.get('default'), _type))
+                    msg=("Argument '%s' in documentation defines default as '%s' "
+                         "but this is incompatible with parameter type '%s'" % (arg, doc_options_arg.get('default'), _type))
                 )
                 continue
 
             if arg_default != doc_default:
                 self.reporter.error(
                     path=self.object_path,
                     code=324,
-                    msg=("argument_spec for '%s' defines default as '%s' but documentation defines default as '%s'" % (arg, arg_default, doc_default))
+                    msg=("Argument '%s' in argument_spec defines default as '%s' "
```

Comment:
Since we are updating this now, let's switch `'%s'` to `%r` for the default values.

Reply:
Fixed.